app/mailers/publisher_mailer.rb

```diff
@@ -208,6 +208,16 @@ def verified_invalid_wallet_internal(publisher, params)
   private
 
   def ensure_fresh_token
-    raise if @publisher.authentication_token.nil? || @publisher.authentication_token_expires_at <= Time.now
+    # Check if we are missing the token and capture to sentry if we are. This should not happen.
+    begin
+      raise "missing token" if @publisher.authentication_token.nil?
+    rescue => e
+      require 'sentry-raven'
```

Comment:
Do we need these lines to capture errors in Sentry? I had always thought when the app loads, it loads Sentry as an exception handler before it gets re-raised even higher.

Reply:
I'm not actually sure. @ayumi set things up like this and I've been following the convention. I would like to know the reasoning, so I'll look into it.